src/broadcast/spectateWorker.ts

```diff
@@ -47,6 +49,10 @@ spectateManager.on(SpectateEvent.NEW_FILE, async (playbackId: string, filePath:
   spectateDetailsSubject.next({ playbackId, filePath });
 });
 
+spectateManager.on(SpectateEvent.RECONNECT, async () => {
+  reconnectSubject.next(1);
```

Comment:
Is there a reason why we return a number here? Does it mean anything?

Because otherwise if the return value means nothing, we can just make it an `Observable<void>` and simply call `.next()`.

Reply:
didn't realize we could do that, will fix.